.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "python.pythonPath": "/usr/local/bin/python3"
```

Comment:
what does this do :o

Reply:
im not sure where that came from but I just deleted it!